pom.xml

```diff
@@ -53,6 +53,7 @@
         <module>connector-j-5</module>
         <module>connector-j-6</module>
         <module>postgres</module>
+        <module>examples/postgres/appengine-standard-java8</module>
```

Comment:
I need to check, but I think this needs to be in a lower pom.

Reply:
Most of the examples don't have entries in the pom. I added it so Intellij would play nice with it, but I can remove it if you prefer.